tests/test.cpp

```diff
@@ -3275,6 +3275,50 @@ void FixedLengthArrayJsonTest(bool binary) {
 #endif
 }
 
+#include "monster_test_bfbs_generated.h" // Generated using --bfbs-comments --bfbs-builtins --cpp --bfbs-gen-embed
```

Comment:
put includes at start with the others?

Reply:
Ok, i'll move it. I placed it here since it a "niche" functionally and wanted it contained in one spot.